README.md

```diff
@@ -1,15 +1,21 @@
 # Kine (Kine is not etcd)
 ==========================
 
-_NOTE: this repository has been recently (2020-11-19) moved out of the github.com/rancher org to github.com/k3s-io
+_NOTE: On 2020-11-19, this repository was moved out of the github.com/rancher org to github.com/k3s-io 
 supporting the [acceptance of K3s as a CNCF sandbox project](https://github.com/cncf/toc/pull/447)_.
 
 ---
 
-Kine is an etcdshim that translates etcd API to sqlite, Postgres, Mysql, and dqlite
+Kine is an etcdshim that translates etcd API to:
+- SQLite
+- Postgres
+- MySQL
+- Dqlite
```

Comment:
we might as well remove dqlite from the list, we haven't built with it in quite a while and I believe most of the repos necessary to build it have been removed by canonical.